### PR TITLE
feat: add discoverable webauthn

### PR DESCRIPTION
**Pull request by jonas-jonas**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

File: driver/config/config.go [hunk 1]
```diff
@@ -1370,6 +1371,10 @@ func (p *Config) WebAuthnForPasswordless(ctx context.Context) bool {
 	return p.GetProvider(ctx).BoolF(ViperKeyWebAuthnPasswordless, false)
 }
 
+func (p *Config) WebAuthnEnableDiscoverableCredentials(ctx context.Context) bool {
+	return p.GetProvider(ctx).BoolF(ViperKeyWebAuthnEnableDiscoverableCredentials, false)
```
Comment (alnr) [inline driver/config/config.go:1375]: This should default to true, no? Is there a downside to enabling this?

Comment (jonas-jonas) [reply driver/config/config.go:1375]: It's changing the UI nodes in the login flow. So I think this should be enabled by the user explicitly. We might want to enable it for new projects in Ory Cloud though.

Comment (kmherrmann) [reply driver/config/config.go:1375]: yes should be default-on in the new experience

File: selfservice/strategy/webauthn/login.go [hunk 2]
```diff
@@ -72,27 +73,62 @@ func (s *Strategy) PopulateLoginMethod(r *http.Request, requestedAAL identity.Au
...
+		loginFlow.UI.SetNode(NewWebAuthnLoginTrigger(string(injectWebAuthnOptions)).WithMetaLabel(text.NewInfoSelfServiceLoginWebAuthn()))
+		loginFlow.UI.Nodes.Upsert(NewWebAuthnLoginInput())
+		loginFlow.UI.Nodes.Upsert(node.NewInputField("discovered", "true", node.WebAuthnGroup, node.InputAttributeTypeHidden))
+
+		identifierNode := loginFlow.UI.Nodes.Find("identifier")
+		if identifierNode != nil {
+			if a, ok := identifierNode.Attributes.(*node.InputAttributes); ok {
+				a.Autocomplete = a.Autocomplete + " webauthn"
+			}
+		}
```
Comment (jonas-jonas) [inline selfservice/strategy/webauthn/login.go:126]: This likely needs a better solution.

We want to add `"webauthn"` to any identifier field shown on the login page, so the browser can show the autofil UI. But with this change, the webauthn strategy no longer renders an `id` input field.

**Pull request closed without merging**